store.js

```diff
@@ -0,0 +1,66 @@
+const { createStore } = require('./redux.min');
+
+const SELECT = 'select';
+const MOVE_UP = 'move_up';
+const MOVE_DOWN = 'move_down';
```

Comment:
```suggestion
const SELECT = 'SELECT';
const MOVE_UP = 'MOVE_UP';
const MOVE_DOWN = 'MOVE_DOWN';
```
camelCase

помести в объект actions